From: Kei Tokunaga <ktokunag@redhat.com>
Subject: [RHEL5.1 PATCH 18/21] dma-mapping.h user our own swiotlb.h header
Date: Thu, 07 Jun 2007 03:46:49 -0400
Bugzilla: 242138
Message-Id: <4667B7E9.4040202@redhat.com>
Changelog: [xen] ia64: use generic swiotlb.h header


bz242138
https://bugzilla.redhat.com/bugzilla/show_bug.cgi?id=242138

Backport of cset#14056.

Need to change to include a generic swiotlb.h for
ia64.  Without this fix, thrid party driver cannot
be compiled.

Thanks,
Kei


rh bug 242138

# HG changeset patch
# User awilliam@xenbuild2.aw
# Date 1171299730 25200
# Node ID 46a8d51134d0603f3db6b6c55ac048afc04839d8
# Parent  dea561992abe678932e2f5dde9346241bf9bc399
[IA64] dma-mapping.h user our own swiotlb.h header

Signed-off-by: Takanori Kasai <kasai.takanori@jp.fujitsu.com>


---

 include/asm-ia64/swiotlb.h                                        |    0 
 linux-2.6.18-21.el5-gerd-order-kei/include/asm-ia64/dma-mapping.h |    2 +-
 2 files changed, 1 insertion(+), 1 deletion(-)

diff -puN include/asm-ia64/dma-mapping.h~14056-IA64_dma-mapping.h_user_our_own_swiotlb.h_header include/asm-ia64/dma-mapping.h
--- linux-2.6.18-21.el5-gerd-order/include/asm-ia64/dma-mapping.h~14056-IA64_dma-mapping.h_user_our_own_swiotlb.h_header	2007-06-07 02:44:33.000000000 -0400
+++ linux-2.6.18-21.el5-gerd-order-kei/include/asm-ia64/dma-mapping.h	2007-06-07 02:44:33.000000000 -0400
@@ -28,7 +28,7 @@
 /* Needed for arch/i386/kernel/swiotlb.c and arch/i386/kernel/pci-dma-xen.c */
 #include <asm/hypervisor.h>
 /* Needed for arch/i386/kernel/swiotlb.c */
-#include <asm-i386/mach-xen/asm/swiotlb.h>
+#include <asm/swiotlb.h>
 
 int dma_map_sg(struct device *hwdev, struct scatterlist *sg, int nents,
                enum dma_data_direction direction);
diff -puN include/asm-ia64/swiotlb.h~14056-IA64_dma-mapping.h_user_our_own_swiotlb.h_header include/asm-ia64/swiotlb.h

_

